frontend/main.js

```diff
@@ -99,7 +99,7 @@ async function startApp () {
 
   // Used in 'chelonia/configure' hooks to emit an error notification.
   function errorNotification (activity: string, error: Error, message: GIMessage) {
-    sbp('gi.notifications/emit', 'CHELONIA_ERROR', { activity, error, message })
+    sbp('gi.notifications/emit', 'CHELONIA_ERROR', { createdDate: new Date().toISOString(), activity, error, message })
```

Comment:
FYI, this notification now runs afoul of `Problem 2 - different hash for the same notifications` because it uses `new Date().toISOString()`.

Meaning: the same notification on different devices will have different hashes.

Reply:
This was my intention to make all the errors independent because
- Same error could occur several times
- Even though it seems to be same, the reason could be different from one to another, from device to device